cli/src/pixl_cli/_database.py

```diff
@@ -101,7 +101,7 @@ def _filter_exported_messages(
 ) -> pd.DataFrame:
     merged = messages_df.merge(
         images_df,
-        on=["accession_number", "mrn", "study_date"],
+        on=["accession_number", "mrn", "study_uid"],
```

Comment:
what would happen here for exported images in the db that have no `study_uid` (or is that even possible)? Would messages for these exported images be sent to the queue?

Reply:
definitely happens with older ones, but they should match by accession number and mrn. The real fun is that for pinpoint alone we've got ones which were populated before we processed the study uid, but I've merged these in the database so that they have the uids already

Reply:
though ironically, I was doing some tidying and had to make good use of a backup in the processed 🤦🏻